### DISCUSSION TITLE
exec /init: exec format error

### DISCUSSION THREAD

**Discussion by yanbert** [tiredofit/docker-nginx-php-fpm#69] [Q&A]
Hello @tiredofit 
thank you for job, 
i use your repo docker-nginx-php-fpm but i got an issiue 

exec /init: exec format error

please do you have any idee about?

**Comment by tiredofit** [comment]
Soudns like you are running it on an architecture that isn't supported.

There _should_ be builds for armv7, armv8 (aarch64), and amd64.

You can check to see if there is in image that works for your architecture by looking at the docker manifest:

`docker manifest inspect docker.io/tiredofit/nginx-php-fpm:8.2-alpine`
